deployment/deployment.yaml.template

```diff
@@ -15,9 +15,6 @@ spec:
       labels:
         app: webhook-server
     spec:
-      securityContext:
```

Comment:
@suyogpatil why did you remove this? Typically, we do not want a webhook server to run as root

Reply:
I remember that after adding this securityContext webhook or some related container was not able to create..its been long time so dont exactly remember but it caused some probelm afaik

Reply:
Hmm.. I just tested with this, and it works. Do you mind removing this from the PR for now? If it causes issues, we can remove it again, with a proper bug description attached.

Reply:
I removed it. Thanks